ckanext/canada/plugins.py

```diff
@@ -578,7 +697,7 @@ def get_actions(self):
             ])
         actions.update({k: disabled_anon_action for k in [
             'current_package_list_with_resources',
-            'user_list',
+            'user_list', #FIXME: required for password reset w/ email
```

Comment:
password resets with email address work now if you merge from master

Reply:
@wardi https://github.com/open-data/ckanext-canada/commit/567a35aefabcd61bfffbf14b6c7bc8cb80518a9e <- this commit?

I will remove the comment for now and merge later on and see if it works for the 2.9 setup.